src/lib/userStorage/UserProfileStorage.js

```diff
@@ -388,17 +395,36 @@ export class UserProfileStorage implements ProfileStorage {
         {},
       )
 
-    return publicProfile
+    const { fullName, smallAvatar } = publicProfile
+
+    logger.info(`getPublicProfile by field <${field}>`, { fullName })
+
+    return { name: fullName, avatar: smallAvatar }
```

Comment:
why? it was already changed to return the whole profile

Reply:
changed in d65c060a3868c252b0641e1aa2f9c28d663e1efa